### DISCUSSION TITLE
update problems

### DISCUSSION THREAD

**Discussion by Liftboy222** [hassio-addons/addon-adguard-home#379] [Q&A] [answered]
On a Rasperry I have adguard running as a plugin for the homeassistant.

Adguard: Version: v0.107.17

I am a beginner with the topic of homeassistant.

The problem is that adguard does not seem to run an automatic update with this version.

Would a way be to make a complete backup of the homeassistant, to uninstall the plugin adguard and then to get the current version with a new installation and then to get the settings back with the restore of the backup?

Greetings
Frank

**Comment by frenck** [comment, marked as answer]
> The problem is that adguard does not seem to run an automatic update with this version.

Updates are automatically offered in Home Assistant, once the add-on ships updates.

> Would a way be to make a complete backup of the homeassistant, to uninstall the plugin adguard and then to get the current version with a new installation and then to get the settings back with the restore of the backup?

? I'm not following this one. As per above, you'll get an update offered for install by Home Assistant once it is available.

../Frenck

**Reply by Liftboy222** [reply]
I have seen that version v0.107.17 has a problem with automatic update. I am still stuck on this version and cannot get an update offer. I'm trying to fix it by deleting the plugin and then installing the latest version. To keep all the settings in adguard I wanted to restore a complete backup of homeassistant. So the small idea.

Frank

**Reply by frenck** [reply]
> I have seen that version v0.107.17 has a problem with automatic update.

I'm not familiar with such an issue. Please file an issue with reproduction scenario in our issue tracker if you want to report such a thing.

**Reply by AlecRust** [reply]
v0.107.17 is currently the latest AdGuard Home version available using this add-on. This is bundled with add-on version [v4.7.5](https://github.com/hassio-addons/addon-adguard-home/releases/tag/v4.7.5).

Add-on version [v4.7.6](https://github.com/hassio-addons/addon-adguard-home/releases/tag/v4.7.6) was released which includes AdGuard Home version v0.107.21. However the [build failed](https://github.com/hassio-addons/addon-adguard-home/actions/runs/3713072695) for this release, so it is not available to update to in HA yet.

**Reply by Liftboy222** [reply]
That means I simply have to wait till the update is working?

**Reply by AlecRust** [reply]
Yes, check for updates within Home Assistant using the button. It is now available.

**Reply by Liftboy222** [reply]
Not till now - I'll try tomorrow.
Thank you very much for your help!

Frank
